testsuite/fmi-cross-check/addResults.py

```diff
@@ -0,0 +1,440 @@
+import os
+import platform
+import sys
+import pandas as pd
+import shutil
+from tempfile import NamedTemporaryFile
+import subprocess
+import sys
+#sys.path.append("/opt/OMSimulator-dev-181/lib")
+from OMSimulator import OMSimulator
+
+debugPrint = False
+ignoreNotCompliantWithLatestRules = True
+
+class HideOutput(object):
+  '''
+  A context manager that block stdout for its scope, usage:
+
+  with HideOutput():
+      os.system('ls -l')
+  '''
+
+  def __init__(self, *args, **kw):
+      sys.stdout.flush()
+      self._origstdout = sys.stdout
+      self._oldstdout_fno = os.dup(sys.stdout.fileno())
+      self._devnull = os.open(os.devnull, os.O_WRONLY)
+
+  def __enter__(self):
+      self._newstdout = os.dup(1)
+      os.dup2(self._devnull, 1)
+      os.close(self._devnull)
+      sys.stdout = os.fdopen(self._newstdout, 'w')
+
+  def __exit__(self, exc_type, exc_val, exc_tb):
+      sys.stdout = self._origstdout
+      sys.stdout.flush()
+      os.dup2(self._oldstdout_fno, 1)
+
+
+def simulate(testFMUDir, resultDir, modelName, fmiType, luaFile):
+  tempDir = os.path.join(resultDir,"temp")
+  intervals = "500"
+
+  # Run lua file with OMSimulator via shell
+  cmd = "OMSimulator \\\n"                                                     \
+      + "    --deleteTempFiles=true \\\n"                                      \
```

Comment:
I think this option should be removed. `True` is the default value anyways, and it is rather a debug flag then anything else. So it might change at some point in the future and then the script will fail.

Reply:
I deleted it and rebased the master branch.